docs/pages/machine-id/getting-started.mdx

```diff
@@ -0,0 +1,241 @@
+---
+title: Machine ID Getting Started Guide
+description: Getting started with Teleport Machine ID
+---
+
+In this getting started guide, you will use Machine ID to create a bot user for
+a machine and use that identity to connect to said machine.
+
+Here's an overview of what you will do:
+
+1. Download and install Teleport (=teleport.version=)
+2. Create bot user
+3. Start Machine ID
+4. Use certificates issued by Machine ID
+
+## Prerequisites
+
+Before using Machine ID, you will need an existing Teleport cluster.
```

Comment:
I think we should make it explicit at this point that "existing Teleport cluster" can include Teleport Cloud, otherwise this is something a user might be wondering now.